### PR TITLE
Higher priority batch for upcoming Release v1.3.0

### LINKED ISSUE #121: Add an example/demo of a Rollbar.NET plug-in for a popular .NET logging library
for example, .NET tracing listener...

### LINKED ISSUE #122: Update packages of the Samples

### LINKED ISSUE #128: Only collect person.id by default for person tracking
Allow the ability to send person.email and person.username, but by default we should only collect person.id going forward.

@akornich This is a top priority.

### LINKED ISSUE #129: Allow IP collection to be easily turned on and off in config
Keep the default IP collection what it is now, but allow the ability to turn IP collection off in the config.

@akornich  This is a top priority.

### LINKED ISSUE #130: Anonymize IP address config option
We need to allow users the option of collecting IP addresses and anonymizing them. Let's anonymize the IPs as described here: https://support.google.com/analytics/answer/2763052

@akornich  This is a top priority.

### LINKED ISSUE #133: Fix unit tests broken by Visual Studio update v15.7.0

### LINKED ISSUE #134: Enhance multi-targeting configuration by introducing NetVariant MSBuild property

### LINKED ISSUE #135: Simplify/clean-up RollbarConfigSection implementation

### PR DESCRIPTION
**Pull request by akornich**
- resolve #128: Only collect person.id by default for person tracking
      - resolve #129: Allow IP collection to be easily turned on and off in config
      - resolve #130: Anonymize IP address config option
      - resolve #135: Simplify/clean-up RollbarConfigSection implementation
      - resolve #134: Enhance multi-targeting configuration by introducing NetVariant MSBuild property
      - resolve #133: Fix unit tests broken by Visual Studio update v15.7.0
      - resolve #121: Add an example/demo of a Rollbar.NET plug-in for a popular .NET logging library
      - resolve #122: Update packages of the Samples

### CODE REVIEW

File: Rollbar/Common/IpAddressUtility.cs [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+            //const int ipv6TotalComponents = 8;
+            const string ipv6Anonymizer = "...";
+            components = exactIpAddress.Split(ipv6CompontsSeparator);
+            if (components != null 
+                && components.Length > 0 //&& components.Length == ipv6CompontsSeparator
+                )
+            {
+                //components[ipv6CompontsSeparator - 1] = ipv6Anonymizer;
+                //return string.Join($"{ipv6CompontsSeparator}", components);
+                const int ipv6CutOffLenght = 12;
```
Comment (rokob) [inline Rollbar/Common/IpAddressUtility.cs]: Not a big deal but `Lenght` should be `Length`

File: Rollbar/Common/IpAddressUtility.cs [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        {
+            if (string.IsNullOrEmpty(exactIpAddress))
+            {
+                return exactIpAddress;
+            }
+
+            string[] components = null;
+
+            // try IPv4 format:
+            const char ipv4CompontsSeparator = '.';
```
Comment (rokob) [inline Rollbar/Common/IpAddressUtility.cs]: `Componts` should probably be `Components`

File: Rollbar/PersonDataCollectionPolicies.cs [hunk 3]
```diff
@@ -0,0 +1,29 @@
+﻿namespace Rollbar
+{
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Converters;
+    using System;
+
+    /// <summary>
+    /// Lists all applicable Person data collection policies.
+    /// </summary>
+    [Flags]
```
Comment (rokob) [inline Rollbar/PersonDataCollectionPolicies.cs]: This is nice

**Comment by rokob** [comment]
This looks good to me. We should add something to the README about these configuration options.

File: Rollbar/Common/IpAddressUtility.cs (hunk 1 above)
Comment (akornich) [reply Rollbar/Common/IpAddressUtility.cs]: oops. fixed.

File: Rollbar/Common/IpAddressUtility.cs (hunk 2 above)
Comment (akornich) [reply Rollbar/Common/IpAddressUtility.cs]: another oops. fixed. thank you!

**Comment by akornich** [comment]
updated readme

**Pull request merged**